main.go

```diff
@@ -100,6 +102,14 @@ func main() {
 		panic(fmt.Errorf("can't use both schema-only and skip-foreign-keys at once. Foreign Key creation can only be skipped when data migration takes place."))
 	}
 
+	if targetDb == conversion.TARGET_SPANGRES {
+		if !(driverName == conversion.PGDUMP || driverName == conversion.POSTGRES) {
+			panic(fmt.Errorf("cannot convert to spangres when source is not postgres target: %s driver: %s", targetDb, driverName))
```

Comment:
Spangres should be capitalized.
And the use of "postgres target" is confusing. How about being more explicit:

"target-db Spangres can only be used with the \"pg_dump\" and \"postgres\" drivers (target-db=%s driver=%s)"